src/main/java/org/kainos/ea/api/ResponsibilityService.java

```diff
@@ -0,0 +1,35 @@
+package org.kainos.ea.api;
+
+import org.kainos.ea.cli.Responsibility;
+import org.kainos.ea.db.DatabaseConnector;
+import org.kainos.ea.db.ResponsibilityDao;
+import org.kainos.ea.exception.FailedToGetResponsibilitiesException;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class ResponsibilityService {
+
+    private final ResponsibilityDao responsibilityDao;
+    private final DatabaseConnector databaseConnector;
+
+    public ResponsibilityService(ResponsibilityDao responsibilityDao, DatabaseConnector databaseConnector) {
+        this.responsibilityDao = responsibilityDao;
+        this.databaseConnector = databaseConnector;
+    }
+
+    public List<Responsibility> getResponsibilitiesForJob(int jobId) throws FailedToGetResponsibilitiesException {
+        try {
+            // Check if jobId exists in the database before calling the dao
+            if (!responsibilityDao.doesJobExist(databaseConnector.getConnection(), jobId)) {
+                throw new FailedToGetResponsibilitiesException();
```

Comment:
this should be a job not found exception

Reply:
JobNotFoundException class created and invoked here